src/styles/partials/components/_utilities.scss

```diff
@@ -0,0 +1,23 @@
+.visuallyhidden {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
+
+.visuallyhidden.hasfocus,
+.visuallyhidden.focusable:active,
+footer,
+.visuallyhidden.focusable:focus,
+footer {
```

Comment:
@tmccoy529 I'm guessing you amde these cahnges, but didn't push them up?

Reply:
must not have but I just pushed again.